PostProcessing/3DPlotParams.ini

```diff
@@ -5,6 +5,7 @@
 hdf5_path = /home/hd/hd_hd/hd_pb293/WS_GRChombo/testing/GRMilijun_ProcaKerrBH_23320250/hdf5
```

Comment:
This looks like an hardcoded path that users won't have access to

Reply:
the *.ini files are parameter files created by the user, as explained [here](https://github.com/ShaunFell/GRBoondi/wiki/Visualization#postprocessing)